pkg/backporter/backporter.go

```diff
@@ -553,30 +558,118 @@ func CreateCloneHandler(client bugzilla.Client, allTargetVersions sets.String, m
 			handleError(w, err, fmt.Sprintf("unable to fetch bug details- Bug#%d", bugID), http.StatusNotFound, endpoint, bugID, m)
 			return
 		}
-		// Create a clone of the bug
-		cloneID, err := client.CloneBug(bug)
+		if !allTargetVersions.Has(req.FormValue("release")) {
+			handleError(w, fmt.Errorf("invalid argument - %s is not a valid TargetRelease, must be one of %v", req.FormValue("release"), allTargetVersions.List()), fmt.Sprintf("invalid argument - %s is not a valid TargetRelease, must be one of %v", req.FormValue("release"), allTargetVersions.List()), http.StatusBadRequest, endpoint, bugID, m)
+			return
+		}
+
+		// Get clones and sort them
+		clones, err := client.GetAllClones(bug)
 		if err != nil {
-			handleError(w, err, "clone creation failed", http.StatusInternalServerError, endpoint, bugID, m)
+			handleError(w, err, fmt.Sprintf("unable to convert \"ID\" parameter from string to int: %s", req.FormValue("ID")), http.StatusBadRequest, req.URL.Path, 0, m)
 			return
 		}
-		targetRelease := bugzilla.BugUpdate{
-			TargetRelease: []string{
-				req.FormValue("release"),
-			},
+		sort.SliceStable(clones, func(i, j int) bool {
+			if len(clones[i].TargetRelease) == 0 && len(clones[j].TargetRelease) == 0 {
+				return false
+			} else if len(clones[i].TargetRelease) == 0 {
+				return true
+			} else if len(clones[j].TargetRelease) == 0 {
+				return false
```

Comment:
Are you use that the normal `<` for strings doesn't have this behavior?

Reply:
I think this check is necessary since TargetRelease is []string not string